### ISSUE TITLE
Implement Missing Level Categories

### ISSUE THREAD

**Issue by turecross321** [LittleBigRefresh/Refresh#67]
Including but not limited to:

- [ ] lbpcool/lbp2cool (lbp1, lbp 2 and lbp vita cool levels)
- [x] thumbs (highest rated)
- [x] mostHearted
- [x] mmpicks (team picks, depends on #68)
- [x] mostUniquePlays
- [x] busiest

**Comment by jvyden** [comment]
lbp2cool and lbpcool can probably be the same algorithm

**Comment by jvyden** [comment]
I'm going to close this in favor of a new issue since Cool Levels is a whole thing.
